modules/aws_eks/aws_eks.py

```diff
@@ -31,6 +32,7 @@ def post_delete(self, module_idx: int) -> None:
         self.cleanup_cloudwatch_log_group(region)
         self.cleanup_dangling_enis(region)
         self.cleanup_security_groups(region)
+        purge_opta_kube_config(layer=self.layer)
```

Comment:
We dont have a post_delete in the GCP GKE processor; I am wondering how the kubectl purge will work there?

Reply:
move to `K8sServiceModuleProcessor.post_delete` to work for all clouds?

Reply:
It's only needed for these two clouds, and they are not part of K8sServiceModuleProcessor